src/ngraph_encapsulate_op.cc

```diff
@@ -34,6 +34,7 @@
 #include "ngraph_mark_for_clustering.h"
 #include "ngraph_utils.h"
 
+#include "ngraph/runtime/cpu/cpu_backend.hpp"
 #include "ngraph/runtime/interpreter/int_backend.hpp"
 
```

Comment:
Is it possible to include backend.hpp than the specific backend files?
https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/backend.hpp

I haven't investigated whether there is a dependency or not.

Reply:
changed to general backend